### PR TITLE
add object detection yolo model based on tfjs

### PR DESCRIPTION
**Pull request by rickycao-qy**

### CODE REVIEW

File: scripts/tfjs-yolo/package.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@pipcook/plugins-tfjs-yolo",
+  "version": "0.0.15",
+  "description": "js version of odps sdk",
```
Comment (FeelyChau) [inline scripts/tfjs-yolo/package.json:4]: Fix description.

File: scripts/tfjs-yolo/pip.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (FeelyChau) [inline scripts/tfjs-yolo/pip.json:1]: I think we should not upload pipeline file.

**Review by FeelyChau — CHANGES_REQUESTED** [review]

File: scripts/tfjs-yolo/pip.json (hunk 2 above)
Comment (rickycao-qy) [reply scripts/tfjs-yolo/pip.json]: done

File: scripts/tfjs-yolo/package.json (hunk 1 above)
Comment (rickycao-qy) [reply scripts/tfjs-yolo/package.json]: done

**Pull request merged**